MoreLinq.Test/StartsWithTest.cs

```diff
@@ -61,19 +61,20 @@ public bool StartsWithWithStrings(string first, string second)
         [Test]
         public void StartsWithReturnsTrueIfBothEmpty()
         {
-            new int[0].StartsWith(new int[0]);
+            Assert.True(new int[0].StartsWith(new int[0]));
         }
 
         [Test]
         public void StartsWithReturnsFalseIfOnlyFirstIsEmpty()
         {
-            new int[0].StartsWith(new[] {1,2,3});
+            Assert.False(new int[0].StartsWith(new[] {1,2,3}));
         }
 
-        [Test]
-        public void StartsWithReturnsFalseIfOnlySecondIsEmpty()
+        [TestCase("", "", Result = true)]
+        [TestCase("1", "", Result = true)]
+        public bool StartsWithReturnsTrueIfSecondIsEmpty(string first, string second)
         {
-            new[] {1,2,3}.StartsWith(new int[0]);
+            return first.StartsWith(second);
```

Comment:
Good catch, I did not notice. Fixing it now. It's getting kind of subtle with IEnumerable<char> and string overloads, but I can only assume the concrete type has precedence.